__tests__/utils.test.js

```diff
@@ -0,0 +1,72 @@
+import { Log, assert, warn } from '../lib/Utils';
+
+xdescribe('Log', () => {
```

Comment:
This test group is skipped because the [part that should be tested for Log](https://github.com/jamiepine/pulse/blob/master/lib/Utils.js#L6) is commented out in master.